src/System.Runtime.Extensions/tests/System/Environment.GetCommandLineArgs.cs

```diff
@@ -78,7 +78,6 @@ public static void RemoteInvoke(string[] args)
         public static int CheckCommandLineArgs(string[] args)
         {
             string[] cmdLineArgs = Environment.GetCommandLineArgs();
-            string cmdLine = Environment.CommandLine;
```

Comment:
Yes, @stephentoub was going to use it to write more test methods but then decided not to write them yet and he forgot to remove this. Given that this is the only thing that doesn't compile in netcoreapp1.0 and that the variable is unused, he was ok with me removing it for the time.